docs/cmd/tkn_customrun.md

```diff
@@ -0,0 +1,31 @@
+## tkn customrun
+
+Manage CustomRuns
+
+***Aliases**: cr,customruns*
```

Comment:
Do we want `cr`? I can be confusing later because we will add `customtask` too, and then we can't have `ct` as it is for `clustertask`, so this will be a bit inconsistent.

I would say let's think of alias, which can work for both `customrun` and `customtask`

Reply:
I think custom task is not resource, its just concept. So may be this is fine to have